packages/ResponsesAccordion/src/components/Responses/Responses.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import responsesStyles from "./Responses.styles";
+
+const responseShape = {
+  heading: PropTypes.node.isRequired,
+  body: PropTypes.node,
+};
+
+const propTypes = {
+  responses: PropTypes.arrayOf(PropTypes.shape(responseShape)),
+};
+
+const defaultProps = {
+  responses: [],
+};
+
+const Response = ({ heading, body }) => {
+  const I18n = useI18n();
+
+  return (
+    <>
```

Comment:
I _think_ you didn't like this syntax before?
(also note it can't be used 100% consistently since you need `<React.Fragment>` if you need a `key`)

Reply:
I think it was Nahum who didn't like it.  I like it though, I keep misspelling fragment with a lowercase f. 
I'm also not that concerned you need to use the long form with a `key` 🤷‍♂️

Reply:
I would prefer to use `<React.Fragment />` is clearly to the developer that is a component and they can search on google about it and find an answer quicker, and even if we use this syntax we can't avoid use `<React.Fragment />` when keys are involved. 

My `giveAFu` is not really high on this, but I don't see a need to introduce a new "code format" in our code that's all.

Reply:
I'm going to leave it, but will follow up on this topic with the team later (probably in a ShopTalk).